packages/app/src/cli/private/plugins/extension.ts

```diff
@@ -1,18 +1,11 @@
-import {UIExtensionSpec} from '../../models/extensions/ui.js'
-import {FunctionSpec} from '../../models/extensions/functions.js'
+import {ExtensionSpecification} from '../../models/extensions/specification.js'
 import {HookReturnPerExtensionPlugin} from '../../public/plugins/extension.js'
 import {Config} from '@oclif/core'
 import {getArrayRejectingUndefined} from '@shopify/cli-kit/common/array'
 import {fanoutHooks} from '@shopify/cli-kit/node/plugins'
 
-export async function loadUIExtensionSpecificiationsFromPlugins(config: Config): Promise<UIExtensionSpec[]> {
+export async function loadUIExtensionSpecificiationsFromPlugins(config: Config): Promise<ExtensionSpecification[]> {
```

Comment:
mm is this still loading only UIExtensions or all extensions? also there is a little typo in the name "specificiations" -> "specifications"